output/watermark-to-event.json

```diff
@@ -1,4 +1,5 @@
 {
+  "/common/destiny2_content/icons/f359d68324ae21522c299983ff1ef9f2.png": 0,
```

Comment:
Not sure why this is here instead of in `watermark-to-season.json`, it appears in the season pass reward defs... @nev-r any help would be great

Reply:
sorted... disregard